cms/djangoapps/contentstore/views/item.py

```diff
@@ -1135,10 +1134,12 @@ def create_xblock_info(xblock, data=None, metadata=None, include_ancestor_info=F
     if is_concise:
         if child_info and len(child_info.get('children', [])) > 0:
             xblock_info['child_info'] = child_info
+        user_partitions = get_user_partition_info(xblock, schemes=['random'], course=course)
```

Comment:
I don't think you can/should assume that every time "is_concise" is True, you only want the random user_partitions. If that's the case, the property is very poorly named.